### ISSUE TITLE
areflexia

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#3246]
Consider adding new term:
areflexia ( a child of abnormal reflex MP:0001961)
DEF: complete absence or loss of neurological reflexes
SYN: absent reflexes
SYN: absence of reflexes
https://www.merriam-webster.com/medical/areflexia, ISBN:9781786842961

See [HP:0001284](http://www.informatics.jax.org/vocab/hp_ontology/HP:0001284)

**Comment by anna-anagnostop** [comment]
Added new term:
areflexia MP:0031122
